src/components/CheckboxAndroid.tsx

```diff
@@ -0,0 +1,126 @@
+// @ts-nocheck
```

Comment:
is this supposed to be here? this turns off the type checking so ideally lets remove